chat.js

```diff
@@ -350,6 +353,38 @@ class CommandContext {
 		}
 		return true;
 	}
+	checkR9k(user, room, message) {
+		let checksum = Chat.md5(toId(message)).substr(0, 16);
```

Comment:
hmm, is this a big enough concern? I can always make the hashes longer, but I will need to decide now.